### PR TITLE
lint: import eslint from v2.x

### PR DESCRIPTION
**Pull request by jonathanong**

### CODE REVIEW

**Comment by PlasmaPower** [comment]
Would I be correct to assume that travis isn't checking eslint? Should this be fixed?

File: test/application.js [hunk 1]
```diff
@@ -27,7 +26,7 @@ describe('app', function(){
     request(app.listen())
     .get('/')
     .end(function(){});
-  })
+  });
 
   it('should not .writeHead when !socket.writable', function(done){
```
Comment (PlasmaPower) [inline test/application.js]: Why isn't there a space before the `{` like in the diffs above (`/lib`)? Is it because this is part of a function call?

Comment (jonathanong) [inline test/application.js]: i just copied whatever was in v2.x. yeah, they should be the same

**Comment by jonathanong** [comment]
yeah, need to add the script to travis

File: test/application.js (hunk 1 above)
Comment (PlasmaPower) [inline test/application.js]: Which are we standardizing on, and shouldn't it be specified in the eslintrc?

**Comment by jonathanong** [comment]
@stevepeak hey how do i switch this over to codecov? i can't figure out why coveralls is a status check and not codecov

**Comment by jonathanong** [comment]
gonna try opening a new PR

**Pull request closed without merging**